google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -273,6 +347,20 @@ def _obtain_ack_deadline(self, maybe_update: bool) -> float:
                     histogram.MIN_ACK_DEADLINE,
                 )
                 self._ack_deadline = min(self._ack_deadline, flow_control_setting)
+
+            # If the user explicitly sets a min ack_deadline, respect it.
+            if self.flow_control.min_duration_per_lease_extension > 0:
+                # The setting in flow control could be too high, adjust if needed.
+                flow_control_setting = min(
```

Comment:
If we see streaming_pull_response.enable_exactly_once_delivery as true and the user hasn't provided a custom min duration, we should update the min duration to 60 seconds. I do not see this happening in this PR? If I am missing it, please point me to where this is happening.

Reply:
Resolving as per another comment.